SpatialGDK/Source/SpatialGDK/Private/EngineClasses/SpatialNetConnection.cpp

```diff
@@ -40,6 +40,12 @@ void USpatialNetConnection::BeginDestroy()
 
 void USpatialNetConnection::CleanUp()
 {
+	if (PlayerControllerEntity != SpatialConstants::INVALID_ENTITY_ID)
+	{
+		UE_LOG(LogSpatialNetConnection, Warning,
```

Comment:
Should this be an error rather than a warning?

Reply:
The warning here is enough in my opinion.
Here is just a check if the order is not right. 
But it does not mean that this is harmful. 
If we saw this warning message, we should be careful with it but not means urgent.

Reply:
For example NWX has a no warnings policy, so if they see this message, what action will they take?

Reply:
I add a warning here just for sure we fixed the ordering issue.
We will not see this log if the problem is solved.

Reply:
If it implies something is broken then I think it should be an error.